x/ethbridge/client/rest/rest.go

```diff
@@ -53,6 +53,15 @@ type burnOrLockEthReq struct {
 	CrosschainFee     sdk.Int      `json:"cross_chain_fee_amount" yaml:"cross_chain_fee_amount"`
 }
 
+type signProphecyReq struct {
+	BaseReq           rest.BaseReq `json:"base_req"`
+	NetworkDescriptor string       `json:"network_descriptor"`
+	CosmosSender      string       `json:"cosmos_sender"`
+	EthereumAddress   string       `json:"ethereum_receiver"`
```

Comment:
If EthereumAddress is the ethereum receiver address let's rename to `EthereumReceiverAddress`, if EthereumAddress is the token address, let's rename to `EthereumTokenAddress`

`EthereumAddress` on its own is not descriptive enough of what it is.

Also, the signProphecyRequest should also have the nonce on it.

Reply:
it is signer's address, will rename as EthereumSignerAddress, also add nonce in the request.